test/parallel/test-unicode-node-options.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+// Flags: --expose-internals
+require('../common');
+const { getOptionValue } = require('internal/options');
+const assert = require('assert');
+const cp = require('child_process');
+
+const expected_redirect_value = 'foó';
+
+if (process.argv.length === 2) {
```

Comment:
Nit: I'd prefer to have explicit check for 'child process' case (i.e. `if (process.argv[2] === 'test') {`) instead to handle possibly different test start args.